build.proj

```diff
@@ -39,11 +39,28 @@
   </Target>
 
   <Target Name="BuildManaged">
-    <ItemGroup>
-      <_BuildMangedProjects Include="src\dirs.proj" />
+    <ItemGroup Condition="'$(DirectoryToBuild)' == ''">
```

Comment:
Was there a specific reason you moved this into build.proj?

Reply:
Dirs.proj was a container before but as tests are handled separately it only handled the managed non-test part of the build. As we don't have such partial containers I simply moved it to build.proj where it better fits IMO.